### PR TITLE
Tagger update

### PR DESCRIPTION
**Pull request by akshaaatt**
- Moved the Overwrite Tags Button to bottom and fixed window input mode in Manifest
- Added Lottie Animation and changed the layout.
- Fixed the visibility criteria
- Managed colors in a better way

### CODE REVIEW

File: app/src/main/res/layout/fragment_directory_picker.xml [hunk 1]
```diff
@@ -1,33 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
-    android:background="@color/app_bg"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (amCap1712) [inline app/src/main/res/layout/fragment_directory_picker.xml:2]: Let's use ConstraintLayout or LinearLayout here. I think we should not use RelativeLayout anywhere in the app. ConstrainLayout is more powerful and flexible. Its more performant than RelativeLayout as well.

**Review by amCap1712 — CHANGES_REQUESTED** [review]
Overall changes look good! I'll also test on my device to confirm it works as expected.

File: app/src/main/res/layout/fragment_directory_picker.xml (hunk 1 above)
Comment (akshaaatt) [reply app/src/main/res/layout/fragment_directory_picker.xml:2]: Sure, will make the changes and let you know.

**Comment by akshaaatt** [comment]
All done :)

**Pull request merged**